### ISSUE TITLE
Angular2 :host selector issues

### ISSUE THREAD

**Issue by cescoferraro** [dlmanning/gulp-sass#464]
Consider the fact that Angular uses :host as a component selector
https://angular.io/docs/ts/latest/guide/component-styles.html

I am using a simple gulp task like

```javascript
var gulp = require('gulp');
var sass = require('gulp-sass');
gulp.task('sass:compile', function () {
	//noinspection JSUnresolvedFunction
	return gulp.src('./test.sass')
		.pipe(sass().on('error', sass.logError))
		.pipe(gulp.dest("./sass"));

});
```

### test.sass
```sass
body
  background-color: green
:host
  background: #000
```
### sass output
```bash
Error: Properties are only allowed within rules, directives, mixin includes, or other properties.
       If ":host" should be a selector, use "\:host" instead.
        on line 3 of test.sass
  Use --trace for backtrace.
```
### gulp-sass output
```css
body {
  background-color: green; }

:host {
  background: #000; }
```

### test.sass
```sass
body
  background-color: green
\:host
  background: #000
```
### sass output
```css
body {
  background-color: green; }

:host {
  background: #000; }
```

### gulp-sass output
```css
body {
  background-color: green; }

\:host {
  background: #000; }
```

**Comment by xzyfer** [comment]
Please file this issue with LibSass.

**Comment by xzyfer** [comment]
This has been fixed in node-sass@v3.7.0
